assets/js/type-defs/checkout.js

```diff
@@ -1,28 +1,28 @@
- * @property {function()} setRedirectUrl                   Dispatches an action
- *                                                         that sets the redirectUrl
- *                                                         to the given value.
- * @property {function()} setHasError                      Dispatches an action
- *                                                         that sets the checkout
- *                                                         status to having an error.
- * @property {function()} clearError                       Dispatches an action
- *                                                         that clears the hasError
- *                                                         status for the checkout.
- * @property {function()} incrementCalculating             Dispatches an action
- *                                                         that increments the calculating
- *                                                         state for checkout by one.
- * @property {function()} decrementCalculating             Dispatches an action
- *                                                         that decrements the calculating
- *                                                         state for checkout by one.
- * @property {function(CartBillingAddress)} setBillingData Dispatches an action
- *                                                         that updates the billing data.
+ * @property {function()} resetCheckout                 Dispatches an action
+ *                                                      that resets the checkout
+ *                                                      to a pristine state.
+ * @property {function()} setRedirectUrl                Dispatches an action
+ *                                                      that sets the redirectUrl
+ *                                                      to the given value.
+ * @property {function()} setHasError                   Dispatches an action
+ *                                                      that sets the checkout
+ *                                                      status to having an error.
+ * @property {function()} clearError                    Dispatches an action
+ *                                                      that clears the hasError
+ *                                                      status for the checkout.
+ * @property {function()} incrementCalculating          Dispatches an action
+ *                                                      that increments the calculating
+ *                                                      state for checkout by one.
+ * @property {function()} decrementCalculating          Dispatches an action
+ *                                                      that decrements the calculating
+ *                                                      state for checkout by one.
+ * @property {function(CartBillingData)} setBillingData Dispatches an action
```

Comment:
While the descriptions _are_ aligned, the variable names are not. they should be aligned as well.  Also when the description alignment means for a narrow column on the 80 column rule, I'll bump up the description to break on the 100 column ruler.